Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM quay.io/pypa/manylinux1_x86_64
+
+RUN /opt/python/cp37-cp37m/bin/pip install poetry
+
+RUN mkdir signal_processing
+ADD . signal_processing
```

Comment:
This will create the directory itself, if missing, or? If so we could probably remove the preceding line.

Reply:
updated.